mlrun/api/migrations/versions/1c954f8cb32d_schedule_last_run_uri.py

```diff
@@ -0,0 +1,26 @@
+"""Schedule last run uri
+
+Revision ID: 1c954f8cb32d
+Revises: f7b5a1a03629
+Create Date: 2020-11-11 09:39:09.551025
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "1c954f8cb32d"
+down_revision = "f7b5a1a03629"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    with op.batch_alter_table("schedules_v2") as batch_op:
+        batch_op.add_column(sa.Column("last_run_uri", sa.String(), nullable=True))
+
+
+def downgrade():
+    with op.batch_alter_table("schedules_v2") as batch_op:
+        batch_op.drop_column("last_run_uri")
```

Comment:
Although it's a very simple migration, it is touching an existing table (that may have data in it)
hence please add a test for the migration simulating the real world scenario - the table has records without this field of course and verifying all is good